cttportfolio.css

```diff
@@ -0,0 +1,941 @@
+	padding: 25px;
+}
+
+.am_img>img,
+.sub_img_div{
+	max-width: 100%;
+	max-height: 100%;
+	border-radius: 7px;
+}
+
+.am_main_img{
+	display: flex;
+	justify-content: center;
+}
+
+
+/*======================================== SKILLS SECTION ========================================*/
+
+.am_sub {
+	margin-top: 50px;
+	align-items:center;
+}
+
+.sub_img_div{
+	height: 100%;
+}
+
+.sub_img {
+	border-radius: 7px;
+	max-width: 100%;
+}
+
+.am_text {
+	background-color: rgba(255, 182, 193, 0.6);
+	width: 70%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: auto 0;
+	padding: 25px
+}
+
+.am_text>p {
+	font-size: 2.0rem;
+}
+
+.am_text>* {
+	margin: 0;
+}
+
+
+/*======================================== PORTFOLIO SECTION ========================================*/
+
+.pf_top {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.pf_main {
+	display: flex;
+	flex-direction: column;
+	height: 1000px;
+}
+
+.project_container {
+	width: 100%;
+	height: 50%;
+	display: flex;
+	justify-content: space-between;
+}
+
+.project2 {
+	align-items: flex-end;
+	margin-top: 100px;
+}
+
+.project_container>* {
+	height: 100%;
+}
+
+.project_img {
+	height: 100%;
+	max-width: 100%;
+	object-fit: cover;
+}
+
+.pf_link {
+	width: 30%;
+	object-fit: cover;
+}
+
+.project_text {
+	text-decoration: none;
+	color: rgb(2, 7, 9);
+	font-size: 2rem;
+	width: 70%;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	border-radius: 5px;
+}
+
+.project_text>* {
+	padding: 20px;
+}
+
+.project_text h5 {
+	margin: 0;
+	align-self: flex-start;
+}
+
+.project1 .project_text {
+	margin-left: 50px;
+}
+
+.project2 .project_text {
+	margin-right: 50px;
+}
+
+.project2 h5 {
+	align-self: flex-end;
+}
+
+.project2 .project_links {
+	align-self: flex-start;
+}
+
+.project_main_links {
+	width: 70%;
+	align-self: flex-end;
+	display: flex;
+	justify-content: space-between;
+	padding-bottom: 2%;
+}
+
+.project_main_links .project_icon_size {
+	width: 8%;
+}
+
+
+/*======================================== SUB PORTFOLIO SECTION ========================================*/
+
+.pf_sub {
+	display: flex;
+	justify-content: space-between;
+	height: 500px;
+}
+
+.pf_sub>a {
+	width: 30%;
+	height: 100%;
+	text-decoration: none;
+}
+
+.pf_sub_link_div {
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.pf_sub_link_div>img {
+	object-fit: cover;
+	height: 100%;
+	max-width: 100%;
+}
+
+.pf_sub h6 {
+	text-decoration: none;
+	font-weight: 600;
+	font-size: 2.0rem;
+	padding: 5%;
+}
+
+
+/*======================================== CONTACT SECTION ========================================*/
+
+.contactme {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	margin-top: 100px;
+}
+
+.contactmetext {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	align-items: center;
+	width: 100%;
+}
+
+.contactmetext>div {
+	margin: 50px 0px;
+	padding: 50px 50px;
```

Comment:
using 2 values that are the same is pointless, just `padding: 50px;` would do the same thing